README.md

```diff
@@ -22,97 +22,9 @@ will return metrics for a HTTP probe against google.com. The `probe_success` met
     docker build -t blackbox_exporter .
     docker run -d -p 9115:9115 --name blackbox_exporter -v `pwd`:/config blackbox_exporter -config.file=/config/blackbox.yml
 
-## Configuration
```

Comment:
You should link to the new file from the readme, otherwise users won't be able to find it